fbw-a380x/src/wasm/systems/a380_systems/src/electrical/mod.rs

```diff
@@ -469,11 +471,10 @@ impl A380RamAirTurbineController {
         rat_and_emer_gen_man_on: &impl EmergencyElectricalRatPushButton,
         emergency_elec_state: &impl EmergencyElectricalState,
     ) {
-        // TODO check actual A380 logic for deployment : solenoid 1 stubbed for now
-        let solenoid_1_should_trigger_deployment_if_powered = false;
+        let solenoid_1_should_trigger_deployment_if_powered = rat_and_emer_gen_man_on.is_pressed();
 
         let solenoid_2_should_trigger_deployment_if_powered =
-            emergency_elec_state.is_in_emergency_elec() || rat_and_emer_gen_man_on.is_pressed();
+            emergency_elec_state.is_in_emergency_elec();
```

Comment:
Actually solenoid 2 is used for manual deployment and solenoid 1 is for automatic deployment.